### ISSUE TITLE
AP0 Gamut with OpenDRT

### ISSUE THREAD

**Issue by antonmeleshkevich** [jedypod/open-display-transform#13]
When primaries set to AP0, OpenDRT gamut compress does this strange black thing with blue colors.

![Screenshot 2021-11-04 052153](https://user-images.githubusercontent.com/34407001/140232321-e0ed0f0a-cf63-4770-8b2f-dea7584c9743.jpg)
![Screenshot 2021-11-04 052210](https://user-images.githubusercontent.com/34407001/140232323-27af0d75-b9a8-485f-b84c-7d931c3136a4.jpg)

**Comment by jedypod** [comment]
Thanks for this post!
I have been meaning to reply for a while so sorry about the delay.

The artifacts you are seeing above in out of gamut blue values are from the perceptual model shitting the sheets a little with chromaticities outside the spectral locus, _and_ from the method I was using for gamut compression.

I have reworked and simplified the gamut compression setup to be a more sensible default. I have also enabled checkboxes for gamut compression and perceptual dechroma on the stock OpenDRT DCTL, because it may be something that users would want to enable or disable depending on their workflow.

The result with the same image and settings are not perfect, but maybe they are a bit improved.
![screenshot_2021-11-26_14-25-06](https://user-images.githubusercontent.com/3261432/143623634-87eba5c2-b9e0-4d5d-943e-474915536270.png)
![screenshot_2021-11-26_14-25-17](https://user-images.githubusercontent.com/3261432/143623643-7b89c3d0-a001-4f7c-b269-eafd4c66cdf4.png)
![screenshot_2021-11-26_14-25-23](https://user-images.githubusercontent.com/3261432/143623654-a957149a-9113-49b6-892f-cb69d0a743e0.png)
